### PR TITLE
Feature/multi profile

### PR DESCRIPTION
**Pull request by collinmcneese**
Allows for the inclusion of multiple Inspec profiles for the Effortless package, specified as directory names with `scaffold_profiles` variable.

Currently when running multiple Effortless Inspec packages on a node, only the latest package to send result data will have results showing in the UI for Automate. To work around this a wrapper profile can be used, but then all controls show up under a potentially monolithic profile.

These changes add the ability to include multiple profiles in a single Effortless Inspec package, each of which will have results populated to Automate as discreet profiles.

The other option right now is to use `scaffolding-chef-benchmark` which has two caveats:
* does not support multiple profiles for Windows currently.
* All additional profiles with `scaffolding-chef-benchmark` must be discreet Effortless Audit packages.  This in itself is not an issue, but adds additional complication of maintaining multiple Effortless Audit packages and making sure that versions are all in alignment.  If a customer has 10+ Effortless Audit packages and the underlying Inspec versions change or package dependencies, this could cause an issue from a dependency conflict if only one of the packages were to be rebuilt for a content update with new underlying versions.  This creates a scenario where in order to update one profile safely, all packages must be rebuilt each time for every promotion to make sure there are no dependency conflicts.

### CODE REVIEW

File: .expeditor/verify.scaffolding-chef-infra.yml [hunk 1]
```diff
@@ -113,6 +122,15 @@ steps:
           privileged: true
     timeout_in_minutes: 60
 
+  - label: "[scaffolding-chef-inspec] :linux: :habicat: Run and test InSpec profile with API changes (multi-profile)"
```
Comment (TheLunaticScripter) [inline .expeditor/verify.scaffolding-chef-infra.yml:125]: You don't need to run each of the test scenarios for this change you only need one test scenario for Linux and one for Windows that uses the multi-profile pattern. It is also totally fine if it's using the default profile set that only has one test and the rest of the variables are default.

**Review by TheLunaticScripter — CHANGES_REQUESTED** [review]
This is great but this changes doesn't need to test all the scenarios again it only really needs to test the multi-profile and single profile works. The single profile is covered by the current set of default tests so just the one muti-profile test is sufficient for this change.

File: .expeditor/verify.scaffolding-chef-infra.yml (hunk 1 above)
Comment (collinmcneese) [reply .expeditor/verify.scaffolding-chef-infra.yml]: All set -- pulled out the tests for multi-profile other than the defaults for linux/windows

**Review by TheLunaticScripter — APPROVED** [review]
This is great thank you for this.

**Pull request merged**